packages/components/src/tour_tip/tour_tip_backdrop.tsx

```diff
@@ -54,7 +56,7 @@ export const TourTipBackdrop = ({
     const backdrop = (
         <div
             onClick={onDismiss}
-            className={'tour-tip__backdrop'}
+            className={`tour-tip__backdrop${transparent ? '_transparent' : ''}`}
```

Comment:
Since this classname is doing a BEM convention, I'd recommend following it for the modifier and going `--transparent`.

Reply:
thanks, done